rustsec-audit-check/action.yaml

```diff
@@ -19,6 +19,7 @@ runs:
       with:
         submodules: false
     - name: Install stable cargo-audit
+      shell: bash
```

Comment:
Looks like this needs the shell specified, https://github.com/mobilecoinofficial/earn/actions/runs/15590501039/job/43909445575
```
Error: mobilecoinofficial/gh-actions/v0/rustsec-audit-check/action.yaml (Line: 21, Col: 7): Required property is missing: shell
```

My understanding is that even for windows bash is available in a CI runner, so this should be "good enough"

Reply:
shell is good practice, but should not be a required field. Maybe its changed.